src/components/builds/ConfigurationWithIssues.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { createStyles, Theme } from '@material-ui/core';
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+import { Alert } from '@material-ui/lab';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    configurationTable: {
+      color: theme.palette.type === 'dark' ? theme.palette.primary.light : theme.palette.primary.dark,
+      background: theme.palette.type === 'dark' ? theme.palette.primary.dark : theme.palette.primary.light,
+      width: '100%',
+      borderSpacing: '0px',
+    },
+    lineNumber: {
+      // An attempt to be consistent in colors with chips[1] backgrounds used on the same builds page
+      //
+      // [1]: https://github.com/mui-org/material-ui/blob/bda3d8541b2f3bea14ed241ae8155239d524a24c/packages/material-ui/src/Chip/Chip.js#L14
+      background: theme.palette.type === 'dark' ? theme.palette.grey[700] : theme.palette.grey[300],
+      fontFamily: 'Monaco, monospace',
+      paddingRight: '5px',
+      width: '1%',
+      minWidth: '40px',
```

Comment:
Let's use `theme.spacing` here. Probably something like this will work?

```json
paddingRight: theme.spacing(1.0),
width: theme.spacing(5.0),
```

The default spacing is `8px` so `theme.spacing(5.0)` will result in the same width by default.

Reply:
See https://github.com/cirruslabs/cirrus-ci-web/pull/351/commits/25491036f88bc9441e647ee4f05cea94cafafd3d.